Fix error on running integration tests: Login profile size exceeds 32 KiB (#99)
kyma-project/test-infra: prow/images/cleaner/README.md

# Cleaner Docker Image

## Overview

> This image contains script which perform cleanup of service account profile in the `kyma-project` project. 

Reviewer: ```suggestion
This image contains the script which performs a cleanup of the service account profile in the `kyma-project` project. 
```

> Script requires environment variable `GOOGLE_APPLICATION_CREDENTIALS` which is a path to service account key.

Reviewer: ```suggestion
The script requires the **GOOGLE_APPLICATION_CREDENTIALS** environment variable which is a path to the service account key.
```

> ## Build

Reviewer: ```suggestion
## Installation
```

Reviewer: I know that in some other README.mds it is "Build" but we try to follow the general template and it states otherwise - "Build "headings will be replaced.

> To build Docker image, run this command:

Reviewer: ```suggestion
To build the Docker image, run this command:
```

# Cleaner Docker Image

## Overview
This image contains the script which performs a cleanup of the service account profile in the `kyma-project` project. 

> The Script requires following environment variables:

Reviewer: ```suggestion
The script requires the following environment variables:
```

- **GOOGLE_APPLICATION_CREDENTIALS** which is a path to the service account key.
- **CLOUDSDK_CORE_PROJECT** which is a Gcloud project name.

## Installation

To build the Docker image, run this command:

```bash

> docker build -t cleaner:<version> .

Reviewer: ```suggestion
docker build -t cleaner:{version} .
```
